.envrc

```diff
@@ -0,0 +1,3 @@
+source_url "https://raw.githubusercontent.com/cachix/devenv/95f329d49a8a5289d31e0982652f7058a189bfca/direnvrc" "sha256-d+8cBpDfDBj41inrADaJt+bDWhOktwslgoP5YiGJ1v0="
+
+use devenv
```

Comment:
no newline? is this even a file?

Reply:
This integrates devenv with https://direnv.net/ so if you have direnv installed you will automatically enter the devenv shell when you navigate to this project's root directory.

Reply:
I'll fix the line ending